torchaudio/models/_wavernn.py

```diff
@@ -1,47 +1,45 @@
+from typing import List
+
+import torch
 from torch import Tensor
 from torch import nn
 
-__all__ = ["_ResBlock", "_MelResNet"]
+__all__ = ["_ResBlock", "_MelResNet", "_Stretch2d", "_UpsampleNetwork", "_WaveRNN"]
 
 
 class _ResBlock(nn.Module):
     r"""This is a ResNet block layer. This layer is based on the paper "Deep Residual Learning
     for Image Recognition". Kaiming He,  Xiangyu Zhang, Shaoqing Ren, Jian Sun. CVPR, 2016.
-    It is a block used in WaveRNN. WaveRNN is based on the paper "Efficient Neural Audio Synthesis".
-    Nal Kalchbrenner, Erich Elsen, Karen Simonyan, Seb Noury, Norman Casagrande, Edward Lockhart,
-    Florian Stimberg, Aaron van den Oord, Sander Dieleman, Koray Kavukcuoglu. arXiv:1802.08435, 2018.
+    It is a block used in WaveRNN.
```

Comment:
In Python, the first line has to be complete and give the summary of what it is.
See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Starting with "This is ..." is lengthy but not adding a value, instead, try to come up with some useful information for first time reader. In this case something like `ResNet block based on "Deep Residual Learning
    for Image Recognition"` would do.

Reply:
The description has been updated. Thanks for the comment and link information.